actors/builtin/miner/miner_test.go

```diff
@@ -3795,6 +3796,12 @@ func (h *actorHarness) preCommitSector(rt *mock.Runtime, params *miner.PreCommit
 		rt.ExpectSend(builtin.StorageMarketActorAddr, builtin.MethodsMarket.VerifyDealsForActivation, &vdParams, big.Zero(), &vdReturn, exitcode.Ok)
 	}
 	st := getState(rt)
+
+	pledgeDelta :=  immediatelyVestingFunds(rt, st).Neg()
```

Comment:
Our habit of stacking the vesting funds in order to pay penalties means we now have to deal with these funds actually vesting after 1 day.

Reply:
Creating a better fee stacking and penalty payment expectation assertion interface feels like the ripest of the low hanging fruits growing on the miner test harness tree. 🥝 🍌 🍒 🍏 🍓